<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametersKNNDynamic</class>
 <widget class="QWidget" name="ParametersKNNDynamic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>metric</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="knnNormSpin">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Power for the Lp norm</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelPower">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>p</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>K</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="knnKspin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Number of Nearest Neighbors to be evaluated
As this is Approximate Nearest Neighbor,
thus the actual number of samples evaluated
might be lower than K</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="knnNormCombo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Distance metric for the evaluation of NN</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>L1 norm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L2 norm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lp norm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L-inf norm</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
